docs/dev/upgrade-opentelemetry.md

```diff
@@ -9,20 +9,22 @@ There is a `make` target to update all OpenTelemetry dependencies and regenerate
 make update-opentelemetry
 ```
 
-Read on for details of the actual details of the target, or skip ahead to [Test](#test) to continue the process.
+<details>
+    <summary>update-opentelemetry target details</summary>
```

Comment:
```suggestion
    <summary>`update-opentelemetry` target details</summary>
```

Reply:
Oops, should have tested this suggestion, sorry. Turns out the `<summary>` tag does not support inner markdown. Sending #161 to fix.